File: Python_Books/算法图解/grokking_algorithms_COPY/09_dynamic_programming/python/01_longest_common_subsequence.py
if word_a[i] == word_b[j]:
  # The letters match.
  cell[i][j] = cell[i-1][j-1] + 1
else:
  # The letters don't match.
  cell[i][j] = max(cell[i-1][j], cell[i][j-1])

File: Python_Books/算法图解/grokking_algorithms_COPY/09_dynamic_programming/lua/01_longest_common_subsequence.lua
if word_a[i] == word_b[j] then
    -- The letters match.
    cell[i][j] = cell[i - 1][j - 1] + 1
else
    -- The letters don't match.
    cell[i][j] = math.max(cell[i - 1][j], cell[i][j - 1])
end